src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -897,14 +668,11 @@ class OperatorStore extends BaseStore {
             while (iterations <= MyAccountStore.CHECK_UPDATES_MAX_ITERATIONS) {
```

Comment:
I think that CHECK_UPDATES_MAX_ITERATIONS should'y be defined in MyAccountStore. It isn't used there at all, and there is not inheritance or extendence relation between MyAccountStore and to whoever uses MyAccountStore.CHECK_UPDATES_MAX_ITERATIONS

Reply:
+